source/common/grpc/common.h

```diff
@@ -36,6 +36,12 @@ class Common {
    */
   static bool hasGrpcContentType(const Http::RequestOrResponseHeaderMap& headers);
 
+  /**
+   * @param headers the headers to parse.
+   * @return bool indicating whether the header is a gRPC request header.
```

Comment:
nit: what is a gRPC request header? Can you clarify? Maybe a more descriptive name also?

Reply:
Sorry super pedantically can we call this `isGrpcRequestHeaders`? (It's confusing to understand if we are operating on a single header or a group of headers)